src/main/java/core/basesyntax/service/operation/SupplyOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
The class implementation should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.